charts/alertmanager/templates/statefulset.yaml

```diff
@@ -18,6 +18,10 @@ spec:
     metadata:
       labels:
         {{- include "alertmanager.selectorLabels" . | nindent 8 }}
+{{- if .Values.podAnnotations }}
+      annotations:
+        {{ toYaml .Values.podAnnotations | nindent 8 }}
```

Comment:
```suggestion
        {{- toYaml .Values.podAnnotations | nindent 8 }}
```

Reply:
If I add the `-` I should replace `nindent` with `indent` ?
What do you prefer ?

Reply:
as the suggestion indicates

Reply:
done